storage/src/mmr/journaled.rs

```diff
@@ -32,7 +32,11 @@ impl<B: Blob, E: RStorage<B> + Metrics, H: Hasher> Storage<H> for Mmr<B, E, H> {
     }
 
     async fn get_node(&self, position: u64) -> Result<Option<H::Digest>, Error> {
-        if position >= self.mem_mmr.oldest_retained_pos() {
+        let oldest_retained_pos = self.mem_mmr.oldest_retained_pos();
+        if oldest_retained_pos.is_none() {
```

Comment:
I think you can do (to avoid the unwrap later):
```
let Some(oldest_retained_pos) = oldest_retained_pos else {
  return Ok(None);
}
```

Reply:
done